Sources/Views/ALKChatCell.swift

```diff
@@ -299,7 +299,11 @@ public final class ALKChatCell: SwipeTableViewCell, Localizable {
         {
             messageLabel.attributedText = attributedText
         } else {
-            messageLabel.text = viewModel.theLastMessage
+            guard  let lastMessage = viewModel.theLastMessage else{
+                messageLabel.text = ""
+                return
```

Comment:
We should not return from this function if the text is not present. We need to execute below code even is the text is not present. You can use ternary operator here to check the text.

Reply:
done